builders/xcm/xtokens.md

```diff
@@ -0,0 +1,244 @@
+---
+title: X-Tokens 
+description:  Learn how to send XC-20s to other chains using the X-Tokens pallet. In addition, the X-Tokens precompile allows you to access core functions via the Ethereum API.
+---
+
+# Using the X-Tokens Pallet
+
+![X-Tokens Precompile Contracts Banner](/images/builders/xcm/xc20/xtokens-banner.png)
+
+## Introduction {: #introduction } 
+
+Building an XCM message for fungible asset transfers is not an easy task. Consequently, there are wrapper functions/pallets that developers can leverage to use XCM features on Polkadot/Kusama.
+
+One example of such wrappers is the [X-Tokens](https://github.com/open-web3-stack/open-runtime-module-library/tree/master/xtokens){target=_blank}  pallet, which provides different methods to transfer fungible assets via XCM.
+
+This guide will show you how to leverage the X-Tokens pallet to send XC-20s from a Moonbeam-based network to other chains in the ecosystem (relay chain/parachains). Moreover, you'll also learn how to use the X-Tokens precompile to perform the same actions via the Ethereum API.
+
+**Developers must understand that sending incorrect XCM messages can result in the loss of funds.** Consequently, it is essential to test XCM features on a TestNet before moving to a production environment.
+
+--8<-- 'text/xcm/general-xcm-definitions.md'
```

Comment:
These definitions seem like they should go into XCM overview and just be linked from here?

Reply:
I've also added this to the overview XCM page, but prefer to have it in both locations